src/migrations/20190829124546-create-role.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Roles', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    label: {
```

Comment:
I thought we agreed to use `user` instead of `unassigned` ? Correct me if I'm wrong please or were there subsequent conclusions on that?

Reply:
Remember, every user that signs up will get the lowest default role anyways. So it's structurally better to have an unassigned there. 
The lowest default roles are "Requester" (company)  and "Team Member" (supplier).

Reply:
Alright, cool